frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -0,0 +1,386 @@
+import { useMemo, type VFC, useState, useEffect } from 'react';
+import useTheme from '@mui/material/styles/useTheme';
+import styled from '@mui/material/styles/styled';
+import { usePageTitle } from 'hooks/usePageTitle';
+import Select from 'component/common/select';
+import Box from '@mui/system/Box';
+import Alert from '@mui/material/Alert';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import {
+    Chart as ChartJS,
+    type ChartOptions,
+    CategoryScale,
+    LinearScale,
+    BarElement,
+    type ChartDataset,
+    Title,
+    Tooltip,
+    Legend,
+    type Chart,
+    type Tick,
+} from 'chart.js';
+
+import { Bar } from 'react-chartjs-2';
+import { useInstanceTrafficMetrics } from 'hooks/api/getters/useInstanceTrafficMetrics/useInstanceTrafficMetrics';
+import type { Theme } from '@mui/material/styles/createTheme';
+import Grid from '@mui/material/Grid';
+import { useUiFlag } from 'hooks/useUiFlag';
+
+type ChartDatasetType = ChartDataset<'bar'>;
+
+type SelectablePeriod = {
+    key: string;
+    dayCount: number;
+    label: string;
+    year: number;
+    month: number;
+};
+
+type EndpointInfo = {
+    label: string;
+    color: string;
+    order: number;
+};
+
+const StyledHeader = styled('h3')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(1),
+    alignItems: 'center',
+    fontSize: theme.fontSizes.bodySize,
+    margin: 0,
+    marginTop: theme.spacing(1),
+    fontWeight: theme.fontWeight.bold,
+}));
+
+const padMonth = (month: number): string => {
+    return month < 10 ? `0${month}` : `${month}`;
+};
```

Comment:
Can be simplified:

const padMonth = (month: number): string => month < 10 ? `0${month}` : `${month}`;

Reply:
Good spot!